### DISCUSSION TITLE
in prep for week 11: second project iteration!

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#12] [Show and tell]
Hi, everyone! You can **use this space to post your 5-minute videos**, along with a link to your most current [public-facing project deliverable](https://benmiller314.github.io/dsam2023fall/projects#palimpsest-of-public-iteration); we'll then be able to use the discussion forum to [write replies](https://benmiller314.github.io/dsam2023fall/projects#presentations-and-peer-review), too.

<div class="alert alert-success"><p>To upload a video, just drag and drop the file onto your "Write" tab. <a href="https://youtube.com/clip/UgkxLpJlcixbtxe4iJdDXwh8nyHj8Oh3-Q3D?si=swQMcoUIptTxbrq7">It's that easy!</a></p></div>

Just so you know (and I hope this puts you further at ease), I will *not* be requiring that you play your video in class while you listen to your own recorded voice and squirm. Instead, because you will likely have to practice to get your presentation to fit in 5 minutes, you'll be well-rehearsed to give your 5-minute presentation live to the class, leaving us plenty of time to have a conversation about your work.

See the [end of week 9's lesson plan](https://benmiller314.github.io/dsam2023fall/plans/week-09#hw) for more detailed guidance, but for this second iteration I'm hoping you can (re)frame your project as _a response to some question or problem_. What answers are you starting to see? How do you know if they're working?

**I expect this will still be partial, work-in-progress, etc.** If you're having trouble doing the above, you can always (instead) talk about why you're having trouble doing the above. But try to lean into the almosts, the felt senses, the murky places, if you can. Trying can sometimes surprise you.

**Comment by chloe-cmu** [comment]
[Thompson_Food Additive Status_2023.10.30.csv](https://github.com/benmiller314/dsam2023fall/files/13193894/Thompson_Food.Additive.Status_2023.10.30.csv)
[DSAM Presentation 2023-10-30.zip](https://github.com/benmiller314/dsam2023fall/files/13193895/DSAM.Presentation.2023-10-30.zip)

**Reply by SidraNArshad** [reply]
Speaking back what I think your project is:
As far as I understand it, you haven’t shifted the focus of your project from the last project check-in that we did. You’re still working with the “Food Additive Status List” and processing the information you are getting from it through OpenRefine to make the data neater, with the ultimate goal to organize the data to make it more understandable/accessible….?
In terms of the public-facing aspect of your project:
I remember you saying in class that you are still considering different options for data visualization
I know that Ben acknowledged this in class, and suggested some sort of bar graph (I remember he specifically said to avoid anything like pie charts) because that would work to showcase differences. 
Questions/comments:
Please forgive my ignorance/incomplete understanding of the process you are working through right now which leaves me unsure as to how to ask questions/leave comments that may benefit you and your project. That being said, I feel like you are at a point in your project that you don’t have a lot of questions, you just need to continue working through your data and figure out how to present it. I have no doubt that you’ll kill it like you’ve done in every DSAM class since the beginning of the semester.

**Comment by Rezaetmd** [comment]
Hi my dear friends from DSAM! Here is my presentation only in a PDF version as the PowerPoint one contains videos and Github doesn't allow uploading more than 15 megabytes.

[Presentation4.pdf](https://github.com/benmiller314/dsam2023fall/files/13199168/Presentation4.pdf)

**Comment by Khushboobhutani** [comment]
https://scalar.usc.edu/works/dsam-project-revised

https://github.com/benmiller314/dsam2023fall/assets/143055269/e520102c-3f76-4772-b517-f1c9a3baed70

**Reply by chloe-cmu** [reply]
In terms of saying your project back to you: you've reconfigured your project with a smaller focus, looking at just documentation of conflict in Sheikh Jarrah and the Al-Aqsa Mosque. 

In terms of concrete suggestions, I think one that would help me navigate the website more clearly is by a central hub page that I could return to. This might not be what you want to do at all, of course - so ignore me if you want to have a really structured path to move through the information. I just mention it because as a reader, I tend to like to be able to retrace my steps a bit if I need to. So, for example, if I'm looking at the timeline of events, but if it makes me think of a question about something I read on the "shifting boundaries" page, I like the ability to go back there directly without having to click through "Another wave of dispossession." So maybe a way you could structure the information is basically in a circle instead of a straight line? Right now it's landing page-shifting boundaries-another wave of dispossession-timeline. But you could consider primarily relying on that structure, but including a link back to the landing page on each particular page and an overview of the pages you'll cover on the landing page itself. 

I also want to reiterate the kind of point I said to Sidra. I don't mean to be repetitive, but I do think that there's some overlap, in that you've also chosen to make a challenging project harder in the pursuit of doing meaningful and important work. You're making a website about representations of an incredibly contested space that's seen lots of violence and suffering since the 1940s. So you've set yourself a challenge in the technical (making the website); the content, and how you represent that content clearly and respectfully. You're doing a great job; just keep taking it one step at a time.

**Reply by amj169** [reply]
Hi Khushboo,

Great work on this important project! I think that the timeline is a really necessary tool in representing the events you’ve included about the conflict. This is especially helpful because it allows you to have a certain degree of control when it comes to guidance for the viewer in terms of where you’d like them to start, like how you chose to begin with the wave of dispossession at Damascus Gate and continue all the way through the events that have unfolded since Al-Aqsa. 

I wonder if there might be a way to enlarge the dates at the top of the timeline to help make that more apparent and easier to navigate. Another thing that may help that we briefly discussed in class would be the possible addition of a table of contents page that is more visible so the viewer can more easily navigate the site and toggle back and forth. 

While I was going through your site, I also noticed a tab labeled “Shifting Boundaries Over the Years” that is linked from the Sheikh Jarrah home page. I wonder if you’ll link the other pages here as well, in addition to the index? I’m thinking the timeline page might be helpful to have here, too. 

I look forward to seeing the final iteration of your work and hope it is something we can all share once it’s complete.

**Comment by SidraNArshad** [comment]
https://github.com/benmiller314/dsam2023fall/assets/142813292/1aea9cbb-1050-44f8-a89e-00304237a3c3

https://scalar.usc.edu/works/dsam-project---blm-activists-/what-it-means-to-be-an-activist?t=1698609894568

**Reply by Rezaetmd** [reply]
I really enjoyed your presentation and authentic questions, so first of all thank you for keeping up the good work! I know that it can be overwhelming when you don't know where to begin with especially if those sources are omitted from the web (Twitter, ...) but I guess including them can emphasize your concerns about those lives and legitimizes working on this project for yourself and audiences as well even more. I was also thinking that you may have an introduction to your project on that webpage and also write about how you as a scholar were drawn to this particular subject: activists and their stories. That would enable the general audience to figure out how they should find the right path to follow among the stories and feel not overwhelmed with the diverse lives of these activist. Since I think when you reiterate about how it all started and where it led to, general audience can understand better how you have gone through those narratives and selected bits and pieces of their lives. I hope my comment be of use here :)

Good luck and look forward to your project!

**Reply by chloe-cmu** [reply]
In terms of saying your project back to you: you're interested in tracking narrative representations of Black Lives Matter activists who have died, both in their own words/personal social media, and also in broader media narratives about their deaths. I think this project is a really interesting and important one, and zooming in on one activist in particular is a good idea. Just writing about Marshawn McCarrel provides structure for the project.  

I got the impression in class that you feel a little bit overwhelmed with this project in some ways, but I don't think you're giving yourself enough credit. You've chosen to do a project that is challenging on multiple levels: the technical (building a website, which is new to you); the content (the deaths of Black Lives Matter activists and then the way the media continues to dehumanize them after their deaths); and your own presentation of the content (presenting your research in a way that is clear while also being respectful to McCarrel's memory. Even if this is not a project you take into a more public sphere, the importance of treating it sensitively is still weighing on your mind.) You've taken an already challenging assignment and decided to do it on hard mode. So, if you can, try to take things one step at a time and give yourself some grace as you figure things out. You're doing a great job, and I'm excited to see where this project goes next.

**Comment by amj169** [comment]
Reminder to start video at 2:00 minutes bc I messed up and don't have it in me to edit it out 

Also, video too large so I'll just play it from my computer. But here is the visual aid and website link. 

[Green Red Colorful Groovy Project Retro Deck Presentation.pdf](https://github.com/benmiller314/dsam2023fall/files/13200515/Green.Red.Colorful.Groovy.Project.Retro.Deck.Presentation.pdf)

https://sites.google.com/d/1RljPXM-aoMi04aLCkWH1blNv2w9DQs9f/p/1kiBeMjB1CKYZ38avAUEljDdBl3hCtRGb/edit

**Reply by SidraNArshad** [reply]
Speaking back what I think your project is:
As I understand it, you are still utilizing the same data, but are making it secondary to your project focus. Instead you’re now focusing on pedagogical strategies that are more targeted to, and better perceived by non-neurotypical learners.
You explained that you found R really difficult to engage with and that inspired you to shift the focus less on the data of PhD stipends, and more on how to use a tool like R to refine this data …?
You are trying to mirror the tools that, ADHD learners for example, use on a day-to-day basis….?
Google Site;
I love the theme you are following for the first page of the site that we saw, in terms of “cooking data”---connecting back to the reading we did a few weeks back. I think this approach is engaging and an informal way for people outside of academia to be able to engage with the content you are presenting. 
Questions/comments:
You talked about incorporating reward-based systems into pedagogy – do you mean suggesting this in your videos, or having some sort of interactive tool in the website that users can use as they follow along with your videos?

**Comment by MustafaKandil** [comment]
https://scalar.usc.edu/works/1-85/index

https://pitt-my.sharepoint.com/:v:/r/personal/muk40_pitt_edu/Documents/video%20for%20DSAM/Screen%20Recording%202023-10-30%20at%202.02.38%20AM%202.mov?csf=1&web=1&e=3Z0H9m&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZyIsInJlZmVycmFsQXBwUGxhdGZvcm0iOiJXZWIiLCJyZWZlcnJhbE1vZGUiOiJ2aWV3In19

**Reply by Rezaetmd** [reply]
Hi Mustafa,
The visualization was wonderful, great job! Also, I’m really jealous that you have so many data already visualized by UNESCO! So, I think you’re exactly where you’re supposed to be and upon finishing some of the details for those cultural heritages on map, you’re good to go! Great job, and look forward to your project!

**Reply by amj169** [reply]
Hi Mustafa,

Great work on your project! I can see some noticeable changes since the last iteration that have made things clearer. I love that you’ve included a section defining intangible cultural heritage right up front so that it is one of the first things the viewer sees when they access the site. This was a main question I had last time when you presented, and I think including that there is really helpful when taking in the information. 

In addition to this change, I think the overall layout of your site is logical and very easy to follow. I like that right after the section where you explain intangible cultural heritage, the next thing the viewer reads is about the importance of safeguarding and why it is necessary. I wonder if you might consider adding even a single sentence about this at the end of the intangible cultural heritage part to help the viewers understand your project’s mission right away while also including your argument about why it is so important to have these cultural histories mapped and represented. 

I look forward to learning more and to seeing your final iteration!
